src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/VBDefineConstantsEncoding.cs

```diff
@@ -7,7 +7,7 @@ namespace Microsoft.VisualStudio.ProjectSystem.Debug
 
     [Export(typeof(INameValuePairListEncoding))]
     [ExportMetadata("Encoding", "NameQuotedValuePairListEncoding")]
-    internal sealed class NameQuotedValuePairListEncoding : INameValuePairListEncoding
+    internal sealed class VBDefineConstantsEncoding : INameValuePairListEncoding
```

Comment:
If single symbol is valid value then it doesn't remain NameValuePair anymore. The interface doesn't align exactly.